src/components/TabList/TabList.tsx

```diff
@@ -27,6 +27,10 @@ const TabList: FC<Props> = (props: Props) => {
     ? (_children as ReactElement<TabProps>[])
     : [_children];
 
+  if (otherProps['aria-labelledby'] === undefined && otherProps['aria-label'] === undefined) {
+    console.warn('MRV2: TabList requires aria-labelledby or aria-label.');
+  }
+
```

Comment:
so empty string fails:
```suggestion
  if (!otherProps['aria-labelledby'] && !otherProps['aria-label']) {
    console.warn('MRV2: TabList requires aria-labelledby or aria-label.');
  }

```
also can we make this a util function? so other components can use this check too

Reply:
Moved!